### DISCUSSION TITLE
autofocus on the URL field?

### DISCUSSION THREAD

**Discussion by gilgongo** [kubero-dev/ladder#25] [Ideas]
Great (and interesting for a number of reasons) project!

Seeing as the only interactive element on the page is the input field, would you consider auto-focusing it? That is, changing it from this:

`<input type="text" id="inputField" name="inputField" class="validate" required>`

to

`<input type="text" id="inputField" name="inputField" class="validate" required autofocus>`

**Comment by mms-gianni** [comment]
Very good idea!

**Reply by mms-gianni** [reply]
v0.0.17 has now autofocus.
